detection_rules/schemas/definitions.py

```diff
@@ -40,7 +40,8 @@
                         'network_traffic',
                         'system',
                         'windows',
-                        'sentinel_one_cloud_funnel']
+                        'sentinel_one_cloud_funnel',
+                        'ti_rapid7_threat_command']
```

Comment:
Does this addition make other detection rules such as `threat_intel_indicator_match*` not eligible to be run on `ti_rapid7_threat_command` integration? 

https://github.com/elastic/detection-rules/blob/2a8ea15fa4bf8e2e06e57c3cac547cdfba539d20/detection_rules/rule.py#L1220

We do want all the 4 indicator match rules named `threat_intel_indicator_match*` to be able to run for this integration.

Reply:
Nope. This addition allows for us to properly identify how to add `related_integrations` data to the prebuilt rule object.